js/classes.js

```diff
@@ -0,0 +1,4 @@
+const classes = [
+    { class: "Fullstack Class", teachers},
```

Comment:
I don't know what should i write for this "teachers". I need to ask it!

Reply:
Kadir, in code reviews, the developer does not make any comments. You have already done your job by writing your code. Don't add any comments except the answers to my comments/questions.